tests/integration/operator-integ-suite-ocp.sh

```diff
@@ -24,14 +24,47 @@ set -u
 # Print commands
 set -x
 
-# deploy operator in OCP
-echo "--------------------------------"
-echo "Check that istio operator is running in OCP"
-echo "--------------------------------"
-"${WD}"/check-operator.sh
-
-# create a Istio Helm Install CP and test httpbin
-echo "--------------------------------"
-echo "Create a IstioHelmInstall Control Plane and test httpbin"
-echo "--------------------------------"
-"${WD}"/control-plane-test.sh
+OPERATOR_NS="${OPERATOR_NS:-istio-operator}"
+OPERATOR_NAME="${OPERATOR_NAME:-istio-operator}"
+CONTROL_PLANE_NS="${CONTROL_PLANE_NS:-default}"
+
+BRANCH="${BRANCH:-maistra-3.0}"
+ISTIO_HELM_INSTALL="https://raw.githubusercontent.com/maistra/istio-operator/${BRANCH}/config/samples/maistra.io_v1_istiohelminstall.yaml"
+
+TIMEOUT="3m"
+
+check_ready() {
+    local NS=$1
```

Comment:
Instead of checking for the pods *and* deployment, why not just check for deployment?

The pods might be `Running` even if they are `1/2`.

The deployment will only be `Available` (or `Ready`, don't recall right now) once all pods are running. So, just by checking the deployment should be enough.

Reply:
Checking rollout status may be best way to check deployment because otherwise checking deployment Available would fail if the deployment has not yet been created at time of check. Thus, for time being checking pod and deployment Available is reliable.

Reply:
My point was: There's no point in checking the running status when the check just does a grep for "Running". The command works even if the pod status is 1/2. Check the logs of the job. That's exactly what happened.

After checking this we also check for the deployment. We could simplify this script by just checking the deployment, which will have the same desired result.

But this can be done as an improvement in another PR. This one is good enough for the time being.